### DISCUSSION TITLE
Fallout Processing

### DISCUSSION THREAD

**Discussion by KrutikaMoorjani** [ONDC-Official/ONDC-FIS-Specifications#19] [Q&A] [answered]
**Sole Prop Flow**

1. In case of eNach digital failures / KYC online failure can the lender follow their standard procedures of processing these cases offline?

2. Is there an overall time limit for processing end to end disbursements ?

**Comment by 92shreyansh** [comment]
@Antriksh-Parmar

**Comment by Antriksh-Parmar** [comment, marked as answer]
@KrutikaMoorjani Please help me with the procedure. You can drop at note at antriksh.parmar@ondc.org
2- Yes. Ideally it should be completed in about 7 minutes
